tools/scripts/platform/xilinx/json2cstringv2

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
```

Comment:
as discussed elsewhere, move this script to the project directory, it contains some project specific stuff that don't belong in tools/scripts

Reply:
You are right. Moved to the project folder.